scipy/odr/tests/test_odr.py

```diff
@@ -495,3 +497,23 @@ def func(par, x):
         sd_ind = out.work_ind['sd']
         assert_array_almost_equal(out.sd_beta,
                                   out.work[sd_ind:sd_ind + len(out.sd_beta)])
+
+    def test_output_file_overwrite(self):
+        """
+        Verify fix for gh-1892
+        """
+        def func(b, x):
+            return b[0] + b[1] * x
+
+        p = Model(func)
+        data = Data(np.arange(10), 12 * np.arange(10))
+        tmp_dir = tempfile.mkdtemp()
+        try:
+            ODR(data, p, beta0=[0.1, 13], errfile=tmp_dir+"/error.dat",
+                rptfile=tmp_dir+"/report.dat").run()
```

Comment:
`+ '/'` isn't quite portable, I think you want `os.path.join` here

other than that, LGTM

Reply:
OK. Thanks. I use `os.path.join` now.